### PR TITLE
update CallVariantsPBCCS with Sniffles2

### PR DESCRIPTION
**Pull request by ewanbroad**
Update CallVariantsPBCCS to call variants across the whole genome with Sniffles2. Sniffles2 only calls variants across the genome and not per-chromosome since it's not clear how to merge .snf files.

### CODE REVIEW

File: wdl/tasks/CallVariantsPBCCS.wdl [hunk 1]
```diff
@@ -268,23 +238,30 @@ workflow CallVariants {
                     is_ccs = true,
                     zones = arbitrary.zones
             }
+
             call VariantUtils.ZipAndIndexVCF as ZipAndIndexPBSV {input: vcf = PBSVslow.vcf }
 
-            call Sniffles.Sniffles as SnifflesSlow {
+            call Sniffles2.SampleSV as sample_snf_slow {
```
Comment (SHuang-Broad) [inline wdl/tasks/CallVariantsPBCCS.wdl:244]: tiny: let's just alias that task here as `Sniffles2SV`

File: wdl/tasks/CallVariantsPBCCS.wdl [hunk 2]
```diff
@@ -268,23 +238,30 @@ workflow CallVariants {
...
+                    vcf = sample_snf_slow.vcf
+            }
         }
     }
 
     output {
-        File? sniffles_vcf = select_first([MergeSnifflesVCFs.vcf, ZipAndIndexSniffles.sortedVCF])
-        File? sniffles_tbi = select_first([MergeSnifflesVCFs.tbi, ZipAndIndexSniffles.tbi])
-
+        File? sniffles_vcf = sample_snf_slow.vcf
```
Comment (SHuang-Broad) [inline wdl/tasks/CallVariantsPBCCS.wdl:261]: We can drop `sniffles_vcf`.

File: wdl/tasks/CallVariantsPBCCS.wdl [hunk 3]
```diff
@@ -268,23 +238,30 @@ workflow CallVariants {
...
                     bam    = bam,
                     bai    = bai,
+                    minsvlen = minsvlen,
+                    sample_id = sample_id,
                     prefix = prefix
             }
-            call Utils.InferSampleName as infer {input: bam = bam, bai = bai}
-            call VariantUtils.FixSnifflesVCF as ZipAndIndexSniffles {input: vcf = SnifflesSlow.vcf, sample_name = infer.sample_name}
+
+            call VariantUtils.ZipAndIndexVCF as ZipAndIndexVCF_slow {
```
Comment (SHuang-Broad) [inline wdl/tasks/CallVariantsPBCCS.wdl:253]: how about aliasing to `ZipAndIndexSnifflesVCF`?

**Review by SHuang-Broad — CHANGES_REQUESTED** [review]
Almost ready!

File: wdl/tasks/CallVariantsPBCCS.wdl (hunk 2 above)
Comment (ewanbroad) [reply wdl/tasks/CallVariantsPBCCS.wdl:261]: File? sniffles_vcf = ZipAndIndexVCF_slow.vcfgz
(rename alias)

**Comment by SHuang-Broad** [comment]
@ewanbroad 
did you mean to close this?

**Pull request closed without merging**